src/Workspaces/Core/Portable/Workspace/AdhocWorkspace.cs

```diff
@@ -33,9 +33,10 @@ public override bool CanApplyChange(ApplyChangesKind feature)
             return true;
         }
 
-        public override bool CanOpenDocuments =>
-                // enables simulation of having documents open.
-                true;
+        /// <summary>
+        /// Returns true, signifiying that you can call the open and close document APIs to add the document into the open document list.
```

Comment:
💡 Would it help to add `seealso` lines after the summary pointing to the open and/or close document APIs?

Reply:
It seemed pretty noisy given there's six of them.